ln.py

```diff
@@ -128,10 +130,42 @@ def list_invoices(self):
             if status == 'paid':
                 pay_index = invoice["pay_index"]
                 amount_received_msat = invoice["amount_received_msat"]
+                amount_received_sats = _msat_to_sat(invoice["amount_received_msat"])
                 paid_at = invoice["paid_at"]
                 paid_date = datetime.datetime.fromtimestamp(paid_at, pytz.timezone('Pacific/Auckland'))
                 payment_preimage = invoice["payment_preimage"]
-            results.append({"paid_date": paid_date, "description": description, "status": status, "amount_msat": amount_msat, "amount_sats": amount_sats, "pay_index": pay_index, "amount_received_msat": amount_received_msat, "payment_preimage": payment_preimage, "bolt11": bolt11, "expires_at": expires_at, "payment_hash": payment_hash, "label": label})
+                results.append({"paid_date": paid_date, "description": description, "status": status, "amount_msat": amount_msat, "amount_sats": amount_sats, "pay_index": pay_index, "amount_received_msat": amount_received_msat, "amount_received_sats": amount_received_sats,"payment_preimage": payment_preimage, "bolt11": bolt11, "expires_at": expires_at, "payment_hash": payment_hash, "label": label})
+        return results
+
+    def list_sendpays(self):
+        results = []
+        result_sendpays = self.instance.listsendpays()
+        for sendpay in result_sendpays["payments"]:
+            payment_hash = sendpay["payment_hash"]
+            status = sendpay["status"]
+            created_at = sendpay["created_at"]
+            amount_sent_msat = sendpay["amount_sent_msat"]
+            amount_sent_sats = _msat_to_sat(amount_sent_msat)
+            amount_msat = sendpay["amount_msat"]
+            amount_sats = _msat_to_sat(amount_msat)
+            destination = sendpay["destination"]
+            label = ""
+            if label:
+                label = sendpay["label"]
+            bolt11 = None
+            if bolt11:
+                bolt11 = sendpay["bolt11"]
+            groupid = None
+            if groupid:
+                groupid = sendpay["groupid"]
+            payment_preimage = None
+            fees_sats = None
+            if status == "complete":
+                paid_at = created_at
+                paid_date = datetime.datetime.fromtimestamp(paid_at, pytz.timezone('Pacific/Auckland'))
```

Comment:
we dont want the timezone hardcoded here, we should use the server timezone

Reply:
removing the pytz.timezone would result on everything being defaulted to UTC.
will most likely need to write a function that would make a os.system to query the timezone being used by the server. need to investigate further.

Reply:
we can set the timezone in the container if we want to

Reply:
trying the following in ln.py:
```
if 'SERVER_TIMEZONE' in os.environ:
    server_timezone = os.environ['SERVER_TIMEZONE']
else:
    server_timezone = "UTC"
```

added a line in .env file for SERVER_TIMEZONE

when checking the list_transactions page, its always coming up as UTC timezone.

ive tried adding app.config in app_core as well. its not working though,

Reply:
changed so this can be configured via env variable